src/js/pensions/config/form.js

```diff
@@ -133,9 +138,48 @@ const formConfig = {
               combatSince911
             }
           }
+        },
+        reserveAndNationalGuard: {
+          path: 'military/reserveAndNationalGuard',
```

Comment:
We normally use kebab case instead of camel case. I think `military/reserve-national-guard` would work.